content/yaml/yaml-getting-started.md

```diff
@@ -131,6 +131,8 @@ workflows:
           - name@example.com
       scripts:
         - echo 'Post-publish script'
+    labels:
+       - My first label name
```

Comment:
we use 2 spaces everywhere and let's provide better name to make it self-descriptive